pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -744,8 +741,10 @@ type BackupSpec struct {
 	Type BackupType `json:"backupType,omitempty"`
 	// StorageProvider configures where and how backups should be stored.
 	StorageProvider `json:",inline"`
-	// StorageClassName is the storage class for backup job's PV.
-	StorageClassName string `json:"storageClassName,omitempty"`
+	// The storageClassName of the persistent volume for Backup data storage.
+	// Optionals: Defaults to the default-storage-class-name set in the tidb-operator or Kubernetes default storage classs.
+	// +optional
+	StorageClassName *string `json:"storageClassName,omitempty"`
```

Comment:
apply the same change from https://github.com/pingcap/tidb-operator/pull/1409 to backup/restore spec